__tests__/app.tests.js

```diff
@@ -0,0 +1,24 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    db.end();
+});
+
+describe('Invalid endpoints', () => {
+    test("404: Responds with an error for endpoints that do not exist", () => {
+        return request(app)
+            .get("/notavalidendpoint")
```

Comment:
When testing for an endpoint that doesn't exist, we should still be putting `api/ ` first before `notavalidendpoint`